rs/crypto/benches/vetkd.rs

```diff
@@ -114,14 +114,15 @@ fn bench_verify_encrypted_key_share<M: Measurement, C: CryptoComponentRng>(
     });
 }
 
-fn bench_combine_encrypted_key_shares_all_n<M: Measurement, C: CryptoComponentRng>(
+fn bench_combine_encrypted_key_shares<M: Measurement, C: CryptoComponentRng>(
     group: &mut BenchmarkGroup<'_, M>,
     config: &NiDkgConfig,
-    subnet_size: usize,
+    name_suffix: &str,
+    num_of_shares_to_combine: usize,
     env: &BTreeMap<NodeId, TempCryptoComponentGeneric<C>>,
     rng: &mut ReproducibleRng,
 ) {
-    group.bench_function("VetKdProtocol::combine_encrypted_key_shares (all n)", |b| {
+    group.bench_function(format!("combine_encrypted_key_shares_{name_suffix}"), |b| {
```

Comment:
Just my personal opinion, but this would be a little clearer to me
```suggestion
    group.bench_function(format!("combine_encrypted_key_shares_{num_of_shares_to_combine}"), |b| {
```

Reply:
Sure, that works too! [Done](https://github.com/dfinity/ic/pull/4593/commits/d5bac00e27da51a0b8e90bc546d7859c9426da56).